bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -35,6 +35,7 @@
 {% endblock %}
 
 {% block content %}
+{% include 'mozorg/developer/includes/sub-nav.html' %}
```

Comment:
Hm with further thought, maybe this does need the same `en` conditional as the global nav (instead of a tag), and this page wouldn't show a subnav unless/until it also gets the global nav. What do you think?